overrides/tag_colour_preview.js

```diff
@@ -0,0 +1,67 @@
+// The following jQuery snippet will add a small square next to the selection
```

Comment:
I **love** what this does, but I'm not sure that this is the best place for it

This is the sort of thing that probably shouldn't live in `overrides/` but should rather just be part of core.  I'm happy to move this to `documents/static/js/colours.js` or something so that it's the default for everyone.  I can't see any reason why we wouldn't want this functionality by default.  If you're cool with that, please modify this PR to place this in `documents/static/js/colours.js` (you'll need to create the `js` directory) and I'll edit the Python code to include the `.js` file directly, bypassing the need for an override.

Reply:
Ok, done.